.zenodo.json

```diff
@@ -0,0 +1,28 @@
+{
+    "creators": [
+        {
+            "affiliation": "University of California, San Francisco",
+            "name": "Axen, Seth",
+            "orcid": "0000-0003-3933-8247"
+        },
+        {
+            "affiliation": "AGH University of Science and Technology",
+            "name": "Baran, Mateusz",
+            "orcid": "0000-0001-9667-5579"
+        },
+        {
+            "affiliation": "Technische Universität Chemnitz",
+            "name": "Bergmann, Ronny",
+            "orcid": "0000-0001-8342-7218"
+        }
+    ],
+    "description": "Manopifolds.jl is a library of Riemannian manifolds in Julia",
```

Comment:
```suggestion
    "description": "Manifolds.jl is a library of Riemannian manifolds in Julia",
```

Reply:
Unless we want to change the name :smile:

Reply:
I had to laugh about that name, but I hope I changed that on master (I also accidentally committed this one here and cherry picked it over); sure will change that; today I just merged master back here to start the next days.